d2bs/kolbot/libs/common/Config.js

```diff
@@ -405,13 +405,18 @@ var Config = {
 		EntranceTP: "Entrance TP up",
 		StarTP: "Star TP up",
 		DiabloMsg: "Diablo",
-		WalkClear: false
+		WalkClear: false,
+		SealOrder: ["vizier", "seis", "infector"],
 	},
 	DiabloHelper: {
 		Wait: 120,
 		Entrance: false,
 		SkipIfBaal: false,
-		SkipTP: false
+		SkipTP: false,
+		OpenSeals: false,
+		SafePrecast: true,
+		SealOrder: ["vizier", "seis", "infector"],
+		RecheckSeals: false,
```

Comment:
How come this is an option on the helper but not on the Diablo itself

Reply:
I was thinking in terms of a "leader" running on his own, that normally runs with helpers. This would save you from having to change your leader's Config to include other seals he normally wouldn't, because he'd always recheck them. But..you're right, it should be an option on both. I'll update Diablo.js